fcrepo-jms/src/main/java/org/fcrepo/jms/AbstractJMSPublisher.java

```diff
@@ -73,6 +74,7 @@
      * @throws JMSException if JMS exception occurred
      */
     @Subscribe
+    @AllowConcurrentEvents
```

Comment:
What affect does this annotation have?

Reply:
https://github.com/google/guava/blob/master/guava/src/com/google/common/eventbus/AllowConcurrentEvents.java

It allows the code to be called from multiple threads.

Reply:
Yes, the name suggested as much. What was happening before? Is this basically a small performance optimization?

Reply:
https://github.com/google/guava/blob/master/guava/src/com/google/common/eventbus/EventBus.java#L71

Previously, it would only process one event at a time. I assume this was an oversight and a misconfiguration. I believe the event code is thread-safe, or am I missing something?

Reply:
Yes, this method should be thread-safe. "Oversight" is probably the reason that the method was not given this annotation before. 
I was just wondering if you had any other reason for making the change... it does not sound like it.

Reply:
Nope! I just saw that it wasn't annotated and added it. :)